ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+
+    1 command(s) successfully processed.
+    csql>
+
+
+..
+
+위 예시를 보면 *cub* 계정 소유의 *tsrv* 서버를 생성 한 상태에서 drop user *cub* 구문이 실패하고 있음을 볼 수 있다.
+이후 *cub.tsrv* 서버를 제거 한 후에는 정상적으로 *cub* 게정을 삭제할 수 있었음을 볼 수 있다.     
+
+
+
+.. _alter-server:
+
+ALTER SERVER
+=============
+
+**ALTER** 구문을 이용하여 서버의 정보를 변경할 수 있다. 대상 서버의 소유자를 변경하거나, HOST, PORT, DBNAME, USER, PASSWOED, PROPERTIES, COMMENT에 대한 정보를 갱신 할 수 있다.  
+
+::
+
+    ALTER SERVER <dblink_server_name> <alter_server_list> ;
+     
+        <dblink_server_name> ::=  [owner_name.] server_name 
+        
+        <alter_server_list> ::=
+                <alter_server_list>, <alter_server_item>
+                | <alter_server_item>						   
+        <alter_server_item> ::= 
+                OWNER TO owner_name
+                | CHANGE <connect_item>
+        <connect_item> ::= 
+                HOST = host_string
+                | PORT = port_number 
+                | DBNAME = db_name 
+                | USER = user_name
+                | PASSWORD = [password_string]
+                | PROPERTIES = [properties_string] 
+                | COMMENT = [server_comment_string]
+                
+
+.. note::
+
+    서버의 소유자 또는 소유 그룹의 멤버만 정보를 변경 할 수 있다. 
+    특별히 **DBA** 또는 **DBA** 의 멤버는 모든 서버 정보를 변경할 수 있다.
+
+.. warning::
+
+    HOST, PORT, DBNAME, USER에 대한 값을 제거하는 갱신을 할 수는 없다.
+
+
+.. _owner_to:
+
+OWNER TO 절
+----------------
+
+**OWNER TO** 절을 사용하여 서버의 소유자를 변경할 수 있다. 
+
+::
+
+    ALTER SERVER  [owner_name.] server_name  OWNER TO new_owner_name ;
+    
+*   *owner_name*: 소유자를 변경할 대상 서버의 소유자 이름을 지정한다.
+*   *server_name*: 소유자를 변경할 대상 서버의 이름을 지정한다.
+*   *new_owner_name*: 새로운 소우자 이름을 지정한다.
+
+.. warning::
+    
+    *   하나의 ALTER SERVER 구문에 OWNER TO 절은 오직 한번만  지정되어야 한다.
+
+
+.. code-block:: sql
+    
+    CREATE SERVER srv1 (HOST='broker-server-name', PORT=3300, DBNAME=demodb, USER=dev1);
+    ALTER SERVER srv1 OWNER TO usr1;    
+    ALTER SERVER usr1.srv1 OWNER TO usr2;    
+
+
+.. _change-server:
+
+CHANGE 절
+----------------
+
+**CHANGE** 절은HOST, PORT, DBNAME, USER, PASSWOED, PROPERTIES, COMMENT 항목의 값을 변경 하는데 사용한다.
+
+.. warning::
+    
+    *   하나의 ALTER SERVER 구문에 CHANGE 절은 콤마(,)로 구분하여 여러개가 나열 될 수 있다. 다만 이때 동일한 항목에 대한 CHANGE절은 오직 하나만 지정되어야 한다.
+    *   ALTER SERVER 구문 수행시 언급이 없었던 항목은 그 값이 초기화 되거나 삭제되는 것이 아니고 기존의 값을 그대로 유지하게 된다. 
+
+::
+
+     ALTER SERVER  [owner_name.] server_name CHANGE <connect_item> [, CHANGE <connect_item>] ... ;
+
+        <connect_item> ::= 
+                HOST = host_string
+                | PORT = port_number 
+                | DBNAME = db_name 
+                | USER = user_name
+                | PASSWORD = [password_string]
+                | PROPERTIES = [properties_string] 
+                | COMMENT = [server_comment_string]
+
+*   *owner_name*: 생성할 서버의 소유자 이름을(user name)지정한다.
+*   *server_name*: 생성할 서버의 이름을 지정한다(최대 254바이트).
+*   *host_string*: 원격 접속 할 DBMS 정보를 가지고 있는 Broker 서버의 호스명 또는 IP 주소이다.
+*   *port_number*: 원격 접속 할 DBMS 정보를 가지고 있는 Broker 서버의 포트 번호이다.

```

Comment:
The term BROKER seems to apply only to CUBRID.

Since it supports connections from other DBMSs, shouldn't we have to use a different terminology?

Reply:
When connecting to another DB, we designate our broker. You will be connected to Oracle, etc. through a designated broker.